src/main/java/de/tum/cit/aet/artemis/communication/web/FaqResource.java

```diff
@@ -115,6 +115,18 @@ public ResponseEntity<FaqDTO> updateFaq(@RequestBody Faq faq, @PathVariable Long
         return ResponseEntity.ok().body(dto);
     }
 
+    /**
+     * @param faq      the faq to be checked *
+     * @param courseId the id of the course the faq belongs to
+     *                     This method throws an exception if a non-instructor in the course tries to set the state of an FAQ to ACCEPTED
+     */
+    private void isAtLeastInstructor(Faq faq, Long courseId) {
```

Comment:
Something like this would be clearer. Otherwise I expect it to just check for instructor status
```suggestion
    private void checkPriviledgeForAcceptedElseThrow(Faq faq, Long courseId) {
```